build.gradle

```diff
@@ -155,11 +155,18 @@ task runPhonenumberAdriana (dependsOn: 'classes', type: JavaExec) {
     main = 'jalau.at18.katas.phonenumber.adriana.Main'
     classpath = sourceSets.main.runtimeClasspath
 }
+
 task runLikesAdriana (dependsOn: 'classes', type: JavaExec) {
     main = 'jalau.at18.katas.likes.adriana.Main'
     classpath = sourceSets.main.runtimeClasspath
 }
 
+task runTeam32048 (dependsOn: 'classes', type: JavaExec) {
+    main = 'jalau.at18.katas.2048.team3.Main'
```

Comment:
package need to be fixed, when this task is run, it fails

Reply:
This problem was fixed to 'jalau.at18.katas**.game**2048.team3.Main'